website/content/api/stocks/_index.md

```diff
@@ -1,6 +1,67 @@
 ---
-title: stocks
-keywords: ""
-excerpt: ""
+title: Introduction to the Stocks SDK
+keywords: "stocks, stock, options, option, call, put, earnings, calendar, how-to, guide, scripts, fundamental, analysis, technical, behavioural, analyst, equity, research, api, sdk, application, python, notebook, jupyter"
+excerpt: "This guide introduces the Stocks menu in the context of the OpenBB SDK."
 geekdocCollapseSection: true
 ---
+
+The <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/stocks/" target="_blank">Stocks menu</a> wraps all of the functions from the OpenBB Terminal application into a powerful SDK, enabling users to work with the data in a flexible environment that can be fully customized to meet the needs of any user. Code completion and contextual help makes the SDK is easy to use. Navigation is very similar to operating the CLI Terminal Application. 
+
+## How to use the Stocks SDK
+
+Start a Jupyter Notebook with as little as one line of code:
+
+`from openbb_terminal.api import openbb`</br>
```

Comment:
Also, can you change this to *openbb_terminal.sdk* and rename your own folder so that it works? At some point this is going to get renamed (@minhhoang1023 when?) and it would be a shame if we need to redo all screenshots and text because of it.

**Unfortunately, this also affects the screenshots you took..**